tailnet/test/peer.go

```diff
@@ -213,6 +239,7 @@ func (p *Peer) handleOneResp() error {
 			default:
 				return xerrors.Errorf("unhandled update kind %s", update.Kind)
 			}
+			p.peerUpdates[id] = append(p.peerUpdates[id], update)
```

Comment:
safer to move this ahead of the `switch` since people might be tempted to `return` from a switch case.